buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.sh

```diff
@@ -108,6 +108,13 @@ check () {
     *.ser)
       trace "Ignoring $1 because it appears to match case parameter expansion *.ser"
       CHECK=0;;
+    # Ignore JFR metadata file
```

Comment:
I'm not sure this comment provides much value, but if you think it should stay, it should be corrected: there are two metadata files (plural), and the sentence should end with a period.